.env.babylon-integration.example

```diff
@@ -62,4 +58,4 @@ STAKING_TIME=10000 # ~70 days
 STAKING_AMOUNT=10000 # 0.0001 BTC
 
 # Finality Explorer Configuration
-NEXT_PUBLIC_FINALITY_GADGET_API_URL=http://11.22.33.44:8545
\ No newline at end of file
+NEXT_PUBLIC_FINALITY_GADGET_API_URL=http://11.22.33.44:18080
```

Comment:
is this b/c the default port is set to 18080 somewhere else? can you point me to that?

Reply:
yes, the FG's default http port is `18080`